{"uuid": "54e4e825-88b4-424d-a374-4ddbc7ac012c", "children": ["7cff5a11-0daa-455a-ad69-97ee65439fad"], "befores": [{"name": "_xunit_setup_method_fixture_TestAvatarUpload", "status": "failed", "statusDetails": {"message": "AssertionError: assert 400 == 200\n +  where 400 = <Response [400]>.status_code\n", "trace": "  File \"/opt/anaconda3/envs/ai-native-kernel/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/anaconda3/envs/ai-native-kernel/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/anaconda3/envs/ai-native-kernel/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/anaconda3/envs/ai-native-kernel/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/anaconda3/envs/ai-native-kernel/lib/python3.12/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/opt/anaconda3/envs/ai-native-kernel/lib/python3.12/site-packages/_pytest/python.py\", line 821, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"/opt/anaconda3/envs/ai-native-kernel/lib/python3.12/site-packages/_pytest/python.py\", line 717, in _call_with_optional_argument\n    func()\n  File \"/Users/zongjunyi/PycharmProjects/ai-native-platform/api/backend_management/user/tests/test_avatar.py\", line 41, in setup_method\n    assert login_response.status_code == 200\n"}, "start": 1753703077812, "stop": 1753703078429}], "start": 1753703077812, "stop": 1753703078433}